campfire-games.html

```diff
@@ -0,0 +1,240 @@
+            <a class="nav-link" href="index.html#sponsors">Sponsors</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="campfire-games.html">Campfire Games</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <a class="mlh-trust-badge" id="mlh-trust-badge"
+      href="https://mlh.io/seasons/2021/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2021-season&utm_content=white"
+      target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2021/mlh-trust-badge-2021-white.svg"
+        alt="Major League Hacking 2021 Hackathon Season"></a>
+  </nav>
+  <!-- End Navigation -->
+
+  <main class="main">
+    <!-- Begin "The Campfire Games" -->
+    <section id="intro" class="container">
+      <!-- set margin here -->
+      <h2 class="section-title">The Campfire Games</h2>
+      <p>
+        For this year's theme, Explore Together, The Campfire Games allow you to learn, grow and build with the Bitcamp Community.
+        You will belong to one of three teams, based on personality and interests, allowing you to meet and work with hackers from around the nation that share your passions.
+        By winning unqiue challenges, attending workshops and participating in mini-events, you'll be racking up points for your team, with the winning group receiving limited edition Bitcamp Apparel.
+        FInd your community, develop your team identity and collaborate on something bigger than yourself. #ExploreTogether
+      </p>
+    </section>
+    <!-- End "The Campfire Games" -->
+
+    <!-- Begin "Teams" -->
+    <section id="teams-container" class="container-team">
+      <section id="teams" class="container">
+        <h2 class="section-title">Teams</h2>
+        <div class="row">
+          <div class="col-md-3">
+            <center>
+              <img class="marshie" src="assets/img/marshie_blue.PNG" alt="Blue marshie">
+            </center>
+          </div>
+          <div class="col-md-9">
+            <p class="right-paragraph">
+              <br></br>
+              The <b>blue team</b> is rational and clear-headed. Their curiosity and intelligence are some of their greatest assets in understanding and exploring the world around them!
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-3 order-md-2">
+            <center>
+              <img class="marshie" src="assets/img/marshie_green.PNG" alt="Green marshie">
+            </center>
+          </div>
+          <div class="col-md-9 order-md-1">
+            <p class = "left-paragraph">
+              <br></br>
+              The <b>green team</b> is adaptable, always growing and thriving in new and challenging circumstances.
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-3">
+            <center>
+              <img class="marshie" src="assets/img/marshie_red.PNG" alt="Red marshie">
```

Comment:
```suggestion
              <img class="marshie" src="assets/img/marshie_red.png" alt="Red marshie">
```
I was getting a bug where the images wouldn't load because you used uppercase `.PNG` instead of lowercase. I would change that for all campfire games images.